src/ripple/rpc/handlers/AccountInfo.cpp

```diff
@@ -125,6 +125,15 @@ doAccountInfo(RPC::JsonContext& context)
         }
         result[jss::account_flags] = std::move(acctFlags);
 
+        // The document states that signer_lists is a bool, however
+        // assigning any string value works. Do not allow this
+        // This for api v2 onwards
```

Comment:
The tiniest of nits: The comment says "... allow this This for ...".  Consider removing the capitalized "This".